kucher/view/main_window/register_view_widget/import_export_dialog.py

```diff
@@ -0,0 +1,302 @@
+                        show_error('Export failed',
+                                   f'Parameter {reg.name} cannot be read.',
+                                   str(ex),
+                                   parent)
+                        _logger.exception(f'Register {reg} could not be read')
+                        return
+
+            yaml.dump(register_yaml, _file)
+            display_sucess_message('Export successful',
+                                   f'Parameters have been successfully exported to:\n{file_name}',
+                                   '',
+                                   parent)
+        finally:
+            _file.close()
+
+    asyncio.get_event_loop().create_task(executor())
+
+
+def import_registers(parent, registers):
+    """
+    Import registers values from a .yml (or .yaml) file.
+    The file must match the following pattern:
+
+    full_register_name:
+    - register_value
+
+    e.g:
+
+    ctl.num_attempts:
+    - 100
+    ctl.pwron_slftst:
+    - false
+    ctl.spinup_durat:
+    - 1.7000000476837158
+
+    It can only contain available registers on the current device, with values within the allowed range. The values
+    must be the right type and dimension. If a register is not mentioned in the file, it will keep its former value.
+    """
+    dialog_box = QWidget()
+    dialog_box.setGeometry(10, 10, 1000, 700)
+    file_name, _ = QFileDialog.getOpenFileName(dialog_box, 'Import configuration file', '',
+                                               'YAML Files (*.yml *.yaml);;All Files (*)')
+    if not file_name:
+        return
+    try:
+        with open(file_name, 'r') as file:
+            imported_registers = yaml.load(file, Loader=yaml.Loader)
+            file.close()
+    except IOError as ex:
+        _logger.exception(f'File {file_name} could not be open')
+        show_error('Import failed',
+                   f'Cannot open {file_name}',
+                   f'Error: {str(ex)}',
+                   parent)
+        return
+
+    except Exception as ex:
+        file.close()
+        _logger.exception(f'File {file_name} could not be parsed')
+        show_error('Import failed',
+                   f'Cannot read {file_name}',
+                   f'Error: {str(ex)}',
+                   parent)
+        return
+
+    if imported_registers:
+        result, detail = check_registers(registers, imported_registers)
+        if result == CheckResult.NO_ERROR:
+            write_registers(parent, file_name, registers, imported_registers)
+        else:
+            show_error_box(result, detail, file_name, parent)
+
+
+def check_registers(registers, imported_registers):
+    try:
+        for reg_check in registers:
+            if reg_check.name in imported_registers:
+                if not (reg_check.mutable and reg_check.persistent):
+                    _logger.exception(f'Import failed: this parameter cannot be modified on this device: {reg_check}')
```

Comment:
`exception` is only for exceptions. Here you should use `error` instead. Background: https://docs.python.org/3/library/logging.html#logging.Logger.exception